CMakeLists.txt

```
@@ -269,6 +269,11 @@ if(SECP256K1_BUILD_CTIME_TESTS)
   unset(msan_enabled)
 endif()
 
+if(NOT SECP256K1_USE_EXTERNAL_DEFAULT_CALLBACKS)
+  include(TryAppendLnkerFlags)
```

```suggestion
  include(TryAppendLinkerFlags)
```
and rename the file?

---

Sure! Sorry for a typo :)

---

Thanks! Done.